File: VB/CustomLayoutAlgorithmSample/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace CustomLayoutAlgorithmSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            treeMap.LayoutAlgorithm = New CustomLayoutAlgorithm()
        End Sub
    End Class
End Namespace

File: CS/CustomLayoutAlgorithmSample/Form1.cs
using System;
using System.Windows.Forms;

namespace CustomLayoutAlgorithmSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            treeMap.LayoutAlgorithm = new CustomLayoutAlgorithm();
        }
    }
}
